README.md

```diff
@@ -104,7 +104,7 @@ jobs:
     - name: 'Checkout Github Action' 
       uses: actions/checkout@master
    
-    - uses: azure/login@v1
+    - uses: azure/actions/login@v1

```

Comment:
The new action resides directly under azure org and should work directly as given in sample